pkg/chart/chart.go

```diff
@@ -175,6 +175,17 @@ func (c *Chart) ValuesFilePathsForCI() []string {
 	return c.ciValuesPaths
 }
 
+// HasCIValuesFile checks whether a given CI values file is present.
+func (c *Chart) HasCIValuesFile(filePath string) bool {
+	fileName := path.Base(filePath)
```

Comment:
We should probably use `filepath.Base` instead.

```suggestion
	fileName := filepath.Base(filePath)
```

Reply:
Good call.